### DISCUSSION TITLE
EMQX cluster on Kubernetes via Operator - what is the proper way to restart the cluster?

### DISCUSSION THREAD

**Discussion by ppawlowski** [emqx/emqx#14171] [Q&A] [answered]
As in the title, I need to run some maintenance on my Kubernetes cluster, and I am forced to restart the EMQX cluster created by the EMQX Operator. What is the correct approach to this? Is a stateful set restart enough, or should I consider something else?

Thank you in advance for any guidance.

**Comment by Rory-Z** [comment, marked as answer]
You can get the current statefulSet of EMQX by `kubectl get emqx ${emqxName} -o json | jq '.status.coreNodesStatus.currentRevision'`
And then, can running this command to restart EMQX cluster `kubectl rollout restart sts ${emqxName}-core-${currentReversion}`
